build.sbt

```diff
@@ -201,7 +201,19 @@ lazy val `mesos-simulation` = (project in file("mesos-simulation"))
     .enablePlugins(GitBranchPrompt, CopyPasteDetector)
     .settings(commonSettings: _*)
     .settings(formatSettings: _*)
-    .dependsOn(marathon % "compile->compile; test->test").configs(IntegrationTest)
+    .dependsOn(marathon % "compile->compile; test->test")
     .settings(
       name := "mesos-simulation"
     )
+
+lazy val benchmark = (project in file("benchmark"))
```

Comment:
sure, I don't think its that useful in the build files, but we should reference how we benchmark somewhere else.